commons/abis.js

```diff
@@ -56,6 +56,11 @@ const OLD_AMB_USER_REQUEST_FOR_SIGNATURE_ABI = [
   {
     anonymous: false,
     inputs: [
+      {
+        indexed: true,
+        name: 'messageId',
+        type: 'bytes32'
+      },
```

Comment:
Why did you modify this ABI?
It is a legacy ABI from the older version of AMB, where events had different signatures:
* `UserRequestForSignature(bytes)` instead of `UserRequestForSignature(bytes32,bytes)`
* `UserRequestForAffirmation(bytes)` instead of `UserRequestForAffirmation(bytes32,bytes)`

Since these signatures are already present in history of the production bridge, we should account for both types of events.